src/info/info_redis/info_redis.c

```diff
@@ -270,3 +281,71 @@ void AddToInfo_RSConfig(RedisModuleInfoCtx *ctx) {
   RedisModule_InfoAddFieldLongLong(ctx, "min_phonetic_term_length",
                                    RSGlobalConfig.minPhoneticTermLen);
 }
+
+// IF the crashing thread worked on a spec, output the spec name
+void AddToInfo_CurrentThread(RedisModuleInfoCtx *ctx) {
+  SpecInfo *specInfo = CurrentThread_TryGetSpecInfo();
+  RedisModule_InfoAddSection(ctx, "current_thread");
+  if (!specInfo) {
+    return;
+  }
+  if (specInfo) {
+    StrongRef strong = WeakRef_Promote(specInfo->specRef);
+    IndexSpec *spec = StrongRef_Get(strong);
+    if (!spec) {
```

Comment:
Would be nice to have some more documentation, for instance, when can this happen (`spec == NULL`)?